scientific_library/contrib/setup.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+#
+#  TheVirtualBrain-Scientific Package. This package holds all simulators, and
+# analysers necessary to run brain-simulations. You can use it stand alone or
+# in conjunction with TheVirtualBrain-Framework Package. See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2020, Baycrest Centre for Geriatric Care ("Baycrest") and others
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this
+# program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+#
+#
+
+import shutil
+import setuptools
+
+TVB_VERSION = "2.0.3"
+
+setuptools.setup(name='tvb-contrib',
+                 version=TVB_VERSION,
+                 packages=setuptools.find_packages(),
+                 include_package_data=True,
+                 install_requires=["tvb-library"],
+                 description='A package with TVB contributed additions to the simulator, useful for scripting.',
+                 license="GPL v3",
+                 author_email='tvb.admin@thevirtualbrain.org',
+                 url='http://www.thevirtualbrain.org',
+                 download_url='https://github.com/the-virtual-brain/tvb-root',
+                 keywords='tvb brain simulator neuroscience contrib')
+
+shutil.rmtree('tvb_contrib.egg-info', True)
```

Comment:
what's the reason for this? does this prevent people from doing
```
pip install -e .
```
or 
```
python setup.py develop
```
?

Reply:
It is meant to just cleanup the current GIT clone.  We might achieve something similar with  .gitignore.
`python setup.py develop`, I haven't tested recently the pip install.
Do you see a problem  here?